modules/ROOT/pages/to-describe-an-asset.adoc

```diff
@@ -4,98 +4,94 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-You can edit an asset's portal; change its name, icon, description, tags, categories, and custom fields; and add portal pages with content.
+You can edit an asset's portal, change its name, icon, description, tags, categories, and custom fields; and add portal pages with content.
```

Comment:
STET original: you must separate list items that have internal commas by using semicolons:
```suggestion
You can edit an asset's portal;
change its name, icon, description, tags, categories, and custom fields; 
and add portal pages with content.
```